app/services/supplier_chooser.rb

```diff
@@ -0,0 +1,14 @@
+class SupplierChooser
```

Comment:
maybe `SupplierSelector` is more appropriate?

actually I am not even sure about `SupplierSelector`, it just takes the first supplier.

Reply:
Selector makes me think of xml/html selections 🤔

Reply:
👍